etc/config/ruby.amazon.properties

```
@@ -0,0 +1,17 @@
+compilers=&ruby
+defaultCompiler=ruby30
+
+group.ruby.compilers=ruby30:ruby27:ruby26:ruby25
+group.ruby.isSemVer=true
+group.ruby.baseName=Ruby
+group.ruby.groupName=Ruby YARV
+group.ruby.compilerType=ruby
+
+compiler.ruby30.semver=3.0.2
+compiler.ruby30.exe=/opt/compiler-explorer/ruby-3.0.2/bin/ruby
+compiler.ruby27.semver=2.7.4
+compiler.ruby27.exe=/opt/compiler-explorer/ruby-2.7.4/bin/ruby
+compiler.ruby26.semver=2.6.8
+compiler.ruby26.exe=/opt/compiler-explorer/ruby-2.6.8/bin/ruby
+compiler.ruby25.semver=2.5.9
+compiler.ruby25.exe=/opt/compiler-explorer/ruby-2.5.9/bin/ruby
```

Looks good! I think it'd be conventional to name the compilers ruby274 for 2.7.4. There could be interest in installing versions like 2.7.3 in the future and this would help to avoid name collision.

```suggestion
defaultCompiler=ruby302

group.ruby.compilers=ruby302:ruby274:ruby268:ruby259
group.ruby.isSemVer=true
group.ruby.baseName=Ruby
group.ruby.groupName=Ruby YARV
group.ruby.compilerType=ruby

compiler.ruby302.semver=3.0.2
compiler.ruby302.exe=/opt/compiler-explorer/ruby-3.0.2/bin/ruby
compiler.ruby274.semver=2.7.4
compiler.ruby274.exe=/opt/compiler-explorer/ruby-2.7.4/bin/ruby
compiler.ruby268.semver=2.6.8
compiler.ruby268.exe=/opt/compiler-explorer/ruby-2.6.8/bin/ruby
compiler.ruby259.semver=2.5.9
compiler.ruby259.exe=/opt/compiler-explorer/ruby-2.5.9/bin/ruby
```